datumaro/datumaro/cli/contexts/project/__init__.py

```diff
@@ -483,15 +493,27 @@ def merge_command(args):
     if dst_dir:
         if not args.overwrite and osp.isdir(dst_dir) and os.listdir(dst_dir):
             raise CliException("Directory '%s' already exists "
-                "(pass --overwrite to force creation)" % dst_dir)
+                "(pass --overwrite to overwrite)" % dst_dir)
+    elif args.strategy == MergeStrategy.merge:
+        dst_dir = generate_next_dir_name('merged-project')
 
     first_dataset = first_project.make_dataset()
-    first_dataset.update(second_project.make_dataset())
-
-    first_dataset.save(save_dir=dst_dir)
+    second_dataset = second_project.make_dataset()
+
+    if args.strategy == MergeStrategy.update:
+        first_dataset.update(second_dataset)
+        merged_dataset = first_dataset
+    elif args.strategy == MergeStrategy.merge:
+        merged_dataset = merge_datasets(first_dataset, second_dataset,
```

Comment:
Codacy found an issue: [Argument 'iou_threshold' passed by position and keyword in function call](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5677001)